gunagala/imager.py

```diff
@@ -1561,18 +1579,19 @@ def make_noiseless_image(self,
         """
```

Comment:
As above, should probably mention in the docstring that you can pass keyword arguments to the `SkyCoord` constructor along with `centre`